src/js/components/Layer/stories/Target.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Box, Button, Grid, Grommet, Layer, Select } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const positions = ['center', 'start', 'end', 'top', 'bottom'];
```

Comment:
I noticed that both `end` and `start` are displaying the layer on the same position of `start`. I might be missing something. we might want to choose right/left.

Reply:
Ah, yes. I got ahead of myself. You must not be using Chrome or Firefox. :)